modules/search/search-service/main.bicep

```diff
@@ -0,0 +1,274 @@
+metadata name = 'Search Services'
+metadata description = 'This module deploys a Search Service.'
+metadata owner = 'Azure/module-maintainers'
+
+// ============== //
+//   Parameters   //
+// ============== //
+
+@description('Required. The name of the Azure Cognitive Search service to create or update. Search service names must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and must be between 2 and 60 characters in length. Search service names must be globally unique since they are part of the service URI (https://<name>.search.windows.net). You cannot change the service name after the service is created.')
+param name string
+
+@description('Optional. Defines the options for how the data plane API of a Search service authenticates requests. Must remain an empty object {} if \'disableLocalAuth\' is set to true.')
+param authOptions object = {}
+
+@description('Optional. The name of logs that will be streamed.')
```

Comment:
suggest moving the diagnostics parameters lower in this list

Reply:
Hi @ahmadabdalla, thanks for the quick response and review. The diagnostic params have been moved down as suggested.